typeform/typeform.py

```diff
@@ -112,14 +115,16 @@ def _build_params(self, form, batch_size):
         page_size = batch_size if batch_size else BATCH_SIZE
         params = {
             'page_size': page_size,
+            'sort': 'landed_at,desc',
+            'completed': 1,  # remove the line if you want to include both
```

Comment:
Include both what?

Reply:
- `'completed': 1` returns only completed responses, which have a list of `answers`.
- `'completed': 0` returns only not completed responses with `answers` as `null`.
- Not specifying  the `completed` parameter at all  returns both types of responses.